pytest.ini

```
@@ -24,8 +25,6 @@ addopts =
     -p no:unraisableexception
     -p no:threadexception
     -m "not mpl_image_compare"
```

```suggestion
    -m "not mpl_image_compare"
    --doctest-ignore-import-errors
```
Turns out we need ths due to `ndcube/utils/sphinx/code_context.py`